src/grunt/helper.js

```diff
@@ -111,34 +111,41 @@ ns.init = function (grunt) {
                 reporters: ['progress', 'coverage'],
 
                 coverageReporter: {
-                    reporters: [{
-                        type: 'text-summary'
-                    }, {
-                        type: 'html',
-                        dir: path.join(process.cwd(), 'coverage'),
-                    }],
+                    reporters: [
+                        {
+                            type: 'text-summary',
+                        },
+                        {
+                            type: 'html',
+                            dir: path.join(process.cwd(), 'coverage'),
+                        },
+                    ],
```

Comment:
Also, I'm not sure I understand the relevance of trailing commas being an error in JSON on this discussion. We're not talking about JSON, we're talking about object definitions. If we were gonna make them be JSON, we'd enforce using quoted strings for keys too, right? I don't think you're proposing that, are you?